docs: add prepend option docs for `addComponentsDir` (#25680)
nuxt/nuxt: docs/3.api/5.kit/5.components.md

  Transpile specified path using build.transpile. If set to `'auto'`, it will set `transpile: true` if `node_modules/` is in path.

> #### `opt`

Reviewer: ```suggestion
#### `opts`
```

**Required**: `false`

- `prepend` (optional)

  **Type**: `boolean`

>   If set to `true`, it will be `unshift` into Nuxt Component Dirs instand of `push`.

Reviewer: borrowing wording from elsewhere in the docs:

```suggestion
  If set to `true`, the directory will be prepended to the array with `unshift()` instead of `push()`.
```
